react/Header/Navigation/Navigation.js

```diff
@@ -79,7 +79,7 @@ Navigation.propTypes = {
   locale: PropTypes.string.isRequired,
   linkRenderer: PropTypes.func.isRequired,
   divider: PropTypes.bool.isRequired,
-  activeTab: PropTypes.oneOf(items.map(({ name }) => name))
+  activeTab: PropTypes.string
```

Comment:
Nope, because if you mark a mobile-only tab as active, you get a proptype warning when it flows through this component.